superset/assets/src/welcome/DashboardTable.jsx

```diff
@@ -18,34 +18,106 @@
  */
 import React from 'react';
 import PropTypes from 'prop-types';
-import { Table, Tr, Td, unsafe } from 'reactable-arc';
-import { SupersetClient } from '@superset-ui/connection';
 import { t } from '@superset-ui/translation';
+import { SupersetClient } from '@superset-ui/connection';
+import moment from 'moment';
+import { debounce } from 'lodash';
+import ListView from 'src/components/ListView/ListView';
+import withToasts from 'src/messageToasts/enhancers/withToasts';
 
-import withToasts from '../messageToasts/enhancers/withToasts';
-import Loading from '../components/Loading';
-import '../../stylesheets/reactable-pagination.less';
-
-const propTypes = {
-  search: PropTypes.string,
-  addDangerToast: PropTypes.func.isRequired,
-};
+const PAGE_SIZE = 25;
 
 class DashboardTable extends React.PureComponent {
-  constructor(props) {
-    super(props);
-    this.state = {
-      dashboards: null,
-    };
+  static propTypes = {
+    addDangerToast: PropTypes.func.isRequired,
+    search: PropTypes.string,
+  };
+
+  state = {
```

Comment:
Is this the same as setting `state` in the constructor?

Reply:
Yes, they are essentially the same thing. The class properties is just sugar for setting variables in constructor. I personally prefer this over using the constructor as the constructor requires a call to `super()` which can sometimes be overlooked.